packages/hostScriptsServices/hostScripts/sensors.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Execute the first command
+output=$(paste <(cat /sys/class/thermal/thermal_zone*/type 2>/dev/null) <(cat /sys/class/thermal/thermal_zone*/temp 2>/dev/null) 2>/dev/null | column -s $'\t' -t | sed 's/\(.\)..$/.\1/')
+echo "$output" | grep -E "x*_pkg_temp" | awk '{print $NF}'
```

Comment:
Shouldn't this be a `return` statement? Check other similar scripts in the repo

Reply:
Other scripts use echo to return values.
Return statement can only return int values